arkouda/random/_generator.py

```diff
@@ -23,9 +28,11 @@ class Generator:
     default_rng : Recommended constructor for `Generator`.
     """
 
-    def __init__(self, seed=None):
+    def __init__(self, name_dict=None, seed=None, state=1):
```

Comment:
I can definitely do that! I didn't initially because `default_rng()` is the preferred constructor on generators, but it can't hurt to include that info

Reply:
absolutely! I'll get right on it

Reply:
done!

Reply:
I appreciate it.  Thanks!